games/devtest/mods/unittests/misc.lua

```diff
@@ -103,3 +103,30 @@ local function test_compress()
 end
 unittests.register("test_compress", test_compress)
 
+local test_on_mapblocks_changed_finishes = {}
+minetest.register_on_mapblocks_changed(function(modified_blocks, modified_block_count)
+	if #test_on_mapblocks_changed_finishes == 0 then return end
+	local finishes = test_on_mapblocks_changed_finishes
+	test_on_mapblocks_changed_finishes = {}
+	for _, finish in ipairs(finishes) do
+		finish(modified_blocks, modified_block_count)
+	end
+end)
+local function test_on_mapblocks_changed(cb)
+	local bp1 = vector.new(12, -5, -3)
+	local bp2 = vector.new(13, -5, -3)
+	local p1 = bp1 * core.MAP_BLOCKSIZE
+	local p2 = bp2 * core.MAP_BLOCKSIZE
+	core.get_voxel_manip(p1, p2):write_to_map(false)
```

Comment:
technically this is a no-op that happens to trigger a block change anyway
I think you rather want to test `add_node` or such